assets/scss/components/_header.scss

```diff
@@ -89,6 +89,16 @@
   vertical-align: middle;
 }
 
+body
+  > div.home-header-mobile
+  > div.home-header-card
+  > div.home-header-buttons
+  > a:nth-child(2)
+  > div
+  > div {
+  vertical-align: unset;
+}
```

Comment:
This approach feels very "duck tape"-y. We should be able to put `vertical-align: unset;` on some class that this div is already using, 

Feel free to make an entirely new class if that's easier

Reply:
Made a new class to override the other `vertical-align` - the other classes that the div uses are used by all the other arrows. I agree it was "duck tape"-y, I was being lazy (；´∀｀)